setup.py

```diff
@@ -142,15 +145,19 @@ def _build_dependency(self, aws_lib):
             '-DBUILD_SHARED_LIBS=OFF',
             '-DCMAKE_BUILD_TYPE={}'.format(build_type),
             '-DBUILD_TESTING=OFF',
-            '-DS2N_NO_PQ_ASM=ON',
         ])
         if self.include_dirs:
-            cmake_args.append('-DCMAKE_INCLUDE_PATH={}'.format(';'.join(self.include_dirs)))
+            cmake_args.append('-DCMAKE_INCLUDE_PATH="{}"'.format(';'.join(self.include_dirs)))
```

Comment:
I've wondered what the right way to quote these is...

subprocess.list2cmdline() will turn this into: `cmake "-DCMAKE_INCLUDE_PATHS=\"my crappy path\"" ...`

Should there be the escaped quotes, within the other quotes, like that?

Reply:
Either way works, I prefer to avoid the escapes in the stuff we have to read, python fixes the quoting before passing it to the shell

Reply:
Yeah, I meant, do we need to add quotes at all? because python will add quotes around the whole argument in the end